.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,63 @@
+---
+name: Prepare Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: 'Branch or tag ref to run the workflow on'
+        required: true
+        default: 'main'
+      version:
+        description: 'The version to release (e.g. 1.2.3). This workflow will automatically perform the required version bumps'
+        required: true
+
+env:
+  RELEASE_VERSION: ${{ inputs.version }}
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}
+
+jobs:
+  create_pr:
+    name: "Bump versions and create PR"
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: elastic/apm-pipeline-library/.github/actions/github-token@current
```

Comment:
All these steps are already set in the `release.yml`

I think a composite action similar to the `maven-goal` could help remove the duplicated code between both action, what do you think?

Maybe the name of the branch could be the name of the github workflow `${{ github.workflow }}`

Reply:
🤔 What about using a reusable workflow?
Ref: https://github.com/elastic/ecs-logging-java/pull/226/commits/531e321ec4cb7643745e39e3dd9a2c2414d18314